flow-typed/npm/remotedev-serialize_vx.x.x.js

```diff
@@ -0,0 +1,50 @@
+// flow-typed signature: bf2231eb3ff430dbd168fd5d3b535a03
+// flow-typed version: <<STUB>>/remotedev-serialize_v0.1.8/flow_v0.92.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'remotedev-serialize'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'remotedev-serialize' {
+  /**
+   * Flowtype definitions for index
+   * Generated by Flowgen from a Typescript Definition
+   * Flowgen v1.10.0
+   */
+
+  declare export type Options = { [key: string]: boolean, ... };
+  declare export type Refs = { [key: string]: any, ... };
+  declare export type DefaultReplacer = (key: string, value: any) => any;
+  declare export type Replacer = (key: string, value: any, replacer: DefaultReplacer) => any;
+  declare export type DefaultReviver = (key: string, value: any) => any;
+  declare export type Reviver = (key: string, value: any, reviver: DefaultReviver) => any;
+  declare export function immutable(
+    immutable: any,
+    refs?: Refs | null,
```

Comment:
I'd like to find a way that this kind of fix won't get automatically clobbered when we, say, upgrade this library. (Or when we just run `yarn update-libdefs`, after upgrading any of our libraries.) How to do that depends on what the workflow is that generated these files in the first place, though. Maybe discuss in a chat thread?

Reply:
Sure! If it can fit in seamlessly enough, perhaps, to pick up [this thread](https://chat.zulip.org/#narrow/stream/48-mobile/topic/flow-typed.20for.20Jest/near/854593)?

Reply:
Sure -- a relevant existing thread is even better than a new one :slightly_smiling_face: